### PR TITLE
Replace EscrowFactory entity with LaunchedEscrow entity

### PR DESCRIPTION
**Pull request by Dzeranov**
`EscrowFactory` entity is replaced with `LaunchedEscrow` entity in order to be able to track and show all created escrows in a human dashboard.

### CODE REVIEW

**Comment by Dzeranov** [comment]
@posix4e, @Jalal-1, please take a look at this.

File: src/mapping/escrowfactory.ts [hunk 1]
```diff
@@ -1,31 +1,15 @@
...
-  // `null` checks allow to create entities on demand
-  if (!entity) {
-    entity = new EscrowFactory(event.transaction.from.toHex())
-
-    // Entity fields can be set using simple assignments
-    entity.count = BigInt.fromI32(0)
-  }
-
-  // BigInt and BigDecimal math are supported
-  entity.count = entity.count + BigInt.fromI32(1)
```
Comment (posix4e) [inline src/mapping/escrowfactory.ts:24]: I thought we have to keep this in mind? Is it always 0?

**Review by posix4e — COMMENTED** [review]
I think we need the count. Did you verify this on the polygon. chain?

File: src/mapping/escrowfactory.ts (hunk 1 above)
Comment (Dzeranov) [reply src/mapping/escrowfactory.ts:24]: Actually, we don't need to keep this count anymore. 
In this implementation we are not aggregating escrows by account created them. One entity will represent one launched escrow.

**Review by posix4e — APPROVED** [review]

**Review by Jalal-1 — APPROVED** [review]

**Pull request merged**